ops/shell/memorystore_shell.sh

```diff
@@ -1,91 +0,0 @@
-#! /bin/bash
```

Comment:
Can we drop this section from the maintainer runbook as well? Maybe with a quick tl;dr of what is cached, and when it'll be cleared?

Reply:
We can actually remove the `Memorystore (redis)` section of the `GAE Firebase Setup` page as well!